frontend/src/components/organization/OrganizationPreviewBody.js

```diff
@@ -0,0 +1,92 @@
+import { Box, CardContent, Tooltip, Typography } from "@material-ui/core";
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import PlaceIcon from "@material-ui/icons/Place";
+
+const useStyles = makeStyles((theme) => {
+  return {
+    bold: {
+      fontWeight: "bold",
+    },
+    locationName: {
+      fontWeight: "bold",
+      whiteSpace: "nowrap",
+      width: "100%",
+      overflow: "hidden",
+      OTextOverflow: "ellipsis",
+      textOverflow: "ellipsis",
+    },
+    infoLink: {
+      display: "flex",
+    },
+    cardIconBox: {
+      width: 40,
+      flex: "0 0 40px",
+      display: "inline-block",
+    },
+    textContent: {
+      fontSize: 14,
+      whiteSpace: "normal",
+    },
+    locationBox: {
+      display: "grid",
+      gridTemplateColumns: "40px min-content",
+      marginBottom: theme.spacing(0.5),
+      justifyContent: "center",
+    },
+    locationNameBox: {
+      maxWidth: "220px",
+      overflow: "hidden",
+    },
+    shortenedSummary: {
+      overflow: "hidden",
+      WebkitBoxOrient: "vertical",
+      display: "-webkit-box",
+    },
+    summaryBox: {
+      overflow: "hidden",
+    },
+    contentWrapper: {
+      padding: 0,
+      display: "grid",
```

Comment:
Generally, we usually work with flexbox rather than grid unless we're displaying a longer list of the same component but using grid is also fine.